_extensions/positconfslides/custom.scss

```diff
@@ -4,96 +4,90 @@
 @import url('https://fonts.googleapis.com/css2?family=Source+Code+Pro:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Barlow+Condensed:wght@300;400');
 
-// css
+$font-family-sans-serif: 'Open Sans', sans-serif !default;
+$font-family-monospace: 'Source Code Pro', monospace !default;
+$presentation-heading-font: "Open Sans", sans-serif !default;
+$presentation-title-slide-text-align: left;
+$presentation-heading-font-weight: 300;
+$code-block-font-size: 0.90em;
 
-#title-slide > h1 {
-  text-align: left;
-  font-weight: 300;
-  color: #404041;
+// colors
+$body-bg:      #FFFFFF;
+$body-color:   #404041;
+$code-color:   #717172;
+$selection-bg: #c290a2;
+$link-color:   #3D6E74;
+
+// dark colors
+$dark-bg-text-color:	#F0F0F0 !default;
+$dark-bg-code-color:	#b1b1b3 !default;
+$dark-bg-link-color:	#FF0000 !default;
+
+/*-- scss:rules --*/
+
+#title-slide .subtitle {
+  font-size: 60px;
+  font-weight: 600;
 }
 
-#title-slide > p {
-  color: #818184;
-  font-size: 65px;
-  text-align: left;
+#title-slide .quarto-title-author-name {
+  font-size: 50px;
   font-weight: 300;
 }
 
+#title-slide .institute {
+  font-weight: 1000;
+}
+
+// dark slides
 div.has-dark-background h2 {
-  color: #eee !important; // someone better at design than me should fix this.
+  color: #F0F0F0 !important;
 }
 
 div.has-dark-background {
-  color: #eee !important; // someone better at design than me should fix this.
+  color: #F0F0F0 !important;
 }
 
 section.has-dark-background > h2 {
   font-weight: 300 !important;
-  color: #eee !important; // someone better at design than me should fix this.
+  color: #F0F0F0 !important;
 }
 
-h2 {
-  font-weight: 300 !important;
-  color: #404041 !important;
+section.has-dark-background a {
+  color: $dark-bg-link-color;
 }
 
-
-
-body {
-  font-weight: 400 !important;
-  color: #404041 !important;
+section.has-dark-background .reveal > blockquote {
```

Comment:
Same as previous comment due to the selector not being the same as in usual revealjs slide. 

Also usually `.reveal` should come first as it is put on the most outside node in HTML for revealjs slide